tsconfig.base.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "lib": ["ES2015", "dom"],
+    "sourceMap": true,
+    "strict": true,
+    "strictFunctionTypes": false,
+    "forceConsistentCasingInFileNames": true,
+    "esModuleInterop": true,
+    "skipDefaultLibCheck": true,
+    "skipLibCheck": true,
+    "typeRoots": ["node_modules/@types", "./typings"],
+    "baseUrl": ".",
+    "paths": {
+      "@fluentui/set-version": ["packages/set-version/src/index.ts"],
+      "@fluentui/react-conformance": ["packages/react-conformance/src/index.ts"],
+      "@fluentui/react-utils": ["packages/react-utils/src/index.ts"],
+      "@fluentui/react-hooks": ["packages/react-hooks/src/index.ts"],
+      "@fluentui/react-make-styles": ["packages/react-make-styles/src/index.ts"],
+      "@fluentui/keyboard-key": ["packages/keyboard-key/src/index.ts"],
+      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"]
+    }
+  },
+  "exclude": ["node_modules"]
```

Comment:
Just curious, why is it necessary to explicitly do this?

Reply:
do what? provide `exclude` ? 
- it's not necessary, but for consistency and smaller git diffs I added it so it can be easily extended by other folders/globs (which we may introduce)

Reply:
Yes the comment was supposed to be specifically on the exclude line. I was just surprised to see this since our configs haven't needed to explicitly exclude node_modules in the past.